app/models/meeting.js

```diff
@@ -108,19 +105,8 @@ export default Model.extend({
     return await agenda.get('agendaName');
   }),
 
-  kindToShow: computed('kind', function() {
-    const options = CONFIG.MINISTERRAAD_TYPES.TYPES;
-    const {
-      kind,
-    } = this;
-    const foundOption = options.find((kindOption) => kindOption.uri === kind);
-
-
-    return EmberObject.create(foundOption);
-  }),
-
   isAnnex: computed('kind', function() {
-    return isAnnexMeetingKind(this.kind);
+    return this.get('kind').get('isAnnexMeeting');
   }),
```

Comment:
Deze computed wordt enkel gebruik in `new-meeting-modal.js`, deze computed weg en lokaal die code doen?

Reply:
Je kan daar gewoon de lokale `!this.isAnnexMeeting` gebruiken lijkt me, we willen gewoon weten wanneer we dat verslag van de vorige vergadering moeten maken.